rpc/server.go

```diff
@@ -130,7 +130,7 @@ func (s *Server) serveRequest(codec ServerCodec, singleShot bool, options CodecO
 
 	defer func() {
 		if err := recover(); err != nil {
-			const size = 64 << 10
+			const size = 128 << 10
```

Comment:
is this necessary? it prints out the runtime stack in case of processing errors, which shouldn't have direct correlation to the payload data size

Reply:
indeed. I did check that http request length is sufficient https://github.com/kaleido-io/quorum/blob/5b9bbd25c080218bb96d692d3574a7a998206adf/rpc/http.go#L35 , so I guess this change is not needed. will back this out